### ISSUE TITLE
False aws_cur_report_definition_invalid_s3_region tflint error

### ISSUE THREAD

**Issue by e-moshaya** [terraform-linters/tflint#898]
I keep getting the following tflint error:

```
Error: "eu-west-2" is an invalid value as s3_region (aws_cur_report_definition_invalid_s3_region)

  on main.tf line 44:
  44:   s3_region = "eu-west-2"
```

The s3 bucket region eu-west-2 (London) is a valid region and should be accepted as a valid region for the s3 bucket region.

```hcl
resource "aws_cur_report_definition" "cost_and_report_definition" {
  provider = aws.us_east_1
  # (Required) Unique name for the report. Must start with a number/letter and is case sensitive. Limited to 256 characters.
  report_name = "cost-analysis-report"
  # (Required) The frequency on which report data are measured and displayed. Valid values are: HOURLY, DAILY.
  time_unit = "HOURLY"
  # (Required) Format for report. Valid values are: textORcsv.
  format = "textORcsv"
  # (Required) Compression format for report. Valid values are: GZIP, ZIP.
  compression = "GZIP"
  # (Required) A list of schema elements. Valid values are: RESOURCES.
  additional_schema_elements = ["RESOURCES"]
  # (Required) Name of the existing S3 bucket to hold generated reports.
  s3_bucket = module.s3_cost_usage_reports.this_s3_bucket_id
  # (Required) Region of the existing S3 bucket to hold generated reports.
  s3_region = "eu-west-2"
  # (Required) A list of additional artifacts. Valid values are: REDSHIFT, QUICKSIGHT.
  additional_artifacts = []
}

## Version

TFLint version 0.19.0
Terraform v0.13.1
+ provider registry.terraform.io/-/aws v3.4.0

**Comment by wata727** [comment]
Thank you for your report. This is a false positive.
It seems that these regions have just been updated recently.
https://github.com/aws/aws-sdk-go/pull/3512/files#diff-3cfde24d82ad886998194d3f5ac75307
